acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/OnChainPrivacyAcceptanceTest.java

```diff
@@ -103,6 +106,96 @@ public void canAddParticipantToGroup() {
     charlie.verify(privateTransactionVerifier.existingPrivateTransactionReceipt(commitmentHash));
   }
 
+  @Test
+  public void aliceCanAddRemoveAddBob() {
+    final String privacyGroupId = createOnChainPrivacyGroup(alice);
+
+    checkOnChainPrivacyGroupExists(privacyGroupId, alice);
+
+    alice.execute(privacyTransactions.privxLockPrivacyGroupAndCheck(privacyGroupId, alice));
+    alice.execute(privacyTransactions.addToPrivacyGroup(privacyGroupId, alice, bob));
+
+    checkOnChainPrivacyGroupExists(privacyGroupId, alice, bob);
+
+    final EventEmitter eventEmitter = deployEventEmitterContract(privacyGroupId, alice);
+
+    final PrivateTransactionReceipt receiptWhileBobMember =
```

Comment:
I don't think `VALUE_WHILE_BOB_MEMBER` and `VALUE_WHILE_BOB_REMOVED` need to be class-level constants. They can be local variables.

Reply:
DONE